sdk/core/Azure.Core/api/Azure.Core.netstandard2.0.cs

```diff
@@ -34,7 +34,9 @@ public partial class AzureKeyCredential
         public override int GetHashCode() { throw null; }
         public static bool operator ==(Azure.ETag left, Azure.ETag right) { throw null; }
         public static bool operator !=(Azure.ETag left, Azure.ETag right) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override string ToString() { throw null; }
+        public string ToString(string format) { throw null; }
```

Comment:
BTW, if we used StandardFormat here, could we default it to "H", i.e.
```csharp
public string ToString(StandardFormat format = 'H');
```
Thought it might not work given we have parameterless ToString.

Reply:
Nah, this does not work.